images/bun/Dockerfile

```
@@ -1,36 +1,30 @@
+
+# Re-declare ARGs after FROM so they are available in the build environment
+ARG BUN_VERSION
+ARG OXLINT_VERSION
+ARG OXFMT_VERSION
 ARG USERNAME=root
 
-# 1. Install packages, generate versions tracker, and install Antigravity in ONE layer
-# Added bash (required for antigravity script), tree, and tmux.
+# 2. Install unpinned OS packages & pinned global tools (using Bun)
+# We completely removed the tool-versions.txt generation block
 RUN --mount=type=cache,target=/var/cache/apk \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)